scoreboard/tests/data.py

```diff
@@ -73,10 +73,10 @@ def make_challenges(cats, tags):
         cat = random.choice(cats)
         #Choose a random subset of tags
         numtags = random.randint(0, len(tags)-1)
-        if not numtags == 0:
-            local_tags = random.sample(tags, numtags)
-        else:
+        if numtags == 0:
             local_tags = []
+        else:
+            local_tags = random.sample(tags, numtags)
```

Comment:
Oh, yeah. I added that in to response to a bug that turned out to be caused by something else, and then didn't think to remove it